bin/getTiming.sh

```diff
@@ -0,0 +1,233 @@
+	"${CFG_DIR}/sharedConfig.cfg"
+	"${HOME}/molgenis.cfg"
+)
+for configFile in "${configFiles[@]}"; do
+	if [[ -f "${configFile}" && -r "${configFile}" ]]; then
+		log4Bash 'DEBUG' "${LINENO}" "${FUNCNAME:-main}" '0' "Sourcing config file ${configFile}..."
+		mixed_stdouterr=$(source ${configFile} 2>&1) || log4Bash 'FATAL' ${LINENO} "${FUNCNAME:-main}" ${?} "Cannot source ${configFile}."
+		source ${configFile}  # May seem redundant, but is a mandatory workaround for some Bash versions.
+	else
+		log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '1' "Config file ${configFile} missing or not accessible."
+	fi
+done
+
+#
+# Execution of this script requires ateambot account.
+#
+if [[ "${ROLE_USER}" != "${DATA_MANAGER}" ]]; then
+	log4Bash 'FATAL' "${LINENO}" "${FUNCNAME:-main}" '1' "This script must be executed by user ${DATA_MANAGER}, but you are ${ROLE_USER} (${REAL_USER})."
+fi
+
+
+workDir="${PRM_ROOT_DIR}"
+projectsDir="${workDir}/projects/"
+logsDir="${workDir}/logs/"
+logsDirSourceServer="/groups/umcg-gd/scr01/logs/"
+
+cd "${projectsDir}"
+find * -nowarn -mtime -40 -type d -maxdepth 0 -exec ls -d {} \; > "${logsDir}/AllProjects40days.txt"
+SAMPLESHEET_SEP=","
+echo -e "unique_id,rawdataname,project,total_min,total_hours,machine,numberofSamples,startTime,finishedTime,copyRawDataToPrmDuration,pipelineDuration,copyProjectDataToPrmTiming,demultiplexingDuration" > ${logsDir}/status_timing.csv
+while read project 
+do
+	echo "start"
+	declare -a sampleSheetColumnNames=()
+        declare -A sampleSheetColumnOffsets=()
+	numberOfSamples=$(ls ${projectsDir}/${project}/run01/results/variants/*.final.vcf.gz | wc -l)
+	sampleSheet="${projectsDir}/${project}/run01/results/${project}.csv"
+
+	IFS="," sampleSheetColumnNames=($(head -1 "${sampleSheet}"))
+	for (( offset = 0 ; offset < ${#sampleSheetColumnNames[@]:-0} ; offset++ ))
+        do
+		columnName="${sampleSheetColumnNames[${offset}]}"
+                sampleSheetColumnOffsets["${columnName}"]="${offset}"
+        done
+
+	sequencingStartDateIndex=$((${sampleSheetColumnOffsets['sequencingStartDate']} + 1))
+	sequencerIndex=$((${sampleSheetColumnOffsets['sequencer']} + 1))
+	runIDIndex=$((${sampleSheetColumnOffsets['run']} + 1))
+	flowcellIndex=$((${sampleSheetColumnOffsets['flowcell']} + 1))
+
+	sequencingStartDate=$(tail -n 1 "${sampleSheet}" | cut -d "${SAMPLESHEET_SEP}" -f ${sequencingStartDateIndex})
+	sequencer=$(tail -n 1 "${sampleSheet}" | cut -d "${SAMPLESHEET_SEP}" -f ${sequencerIndex})
+	runID=$(tail -n 1 "${sampleSheet}" | cut -d "${SAMPLESHEET_SEP}" -f ${runIDIndex})
+	flowcell=$(tail -n 1 "${sampleSheet}" | cut -d "${SAMPLESHEET_SEP}" -f ${flowcellIndex})
+
+	echo "${sequencingStartDate}_${sequencer}_${runID}_${flowcell}"
+	filePrefix="${sequencingStartDate}_${sequencer}_${runID}_${flowcell}"
+	if [[ ${sequencer} == N* || ${sequencer} == M* ]]
+	then
+		if ssh ${sourceServerFQDNprimary} -n test -e "${logsDirSourceServer}/${filePrefix}/run01.demultiplexing.finished"
```

Comment:
de sourceServerFQDNprimary wordt niet mee gegeven in  command line arguments. of word dit op coenzym gedraaid?